tests/manage/z_cluster/nodes/test_disk_failures.py

```diff
@@ -104,7 +104,6 @@ def init_sanity(self):
         """
         self.sanity_helpers = Sanity()
 
-    @aws_platform_required
```

Comment:
I guess we should replace this with something like `cloud_platform_required`. It still won't work on non cloud platforms like bare metal.

Reply:
Done !

Reply:
i have separated the modifications of test_disk_failures from this PR to contain only infra changes